VideoWeb/VideoWeb.AcceptanceTests/Features/JudgePrivateConsultation.feature

```diff
@@ -14,7 +14,6 @@ Scenario: Judge Can Enter The Consultation Room
 Scenario: Panel Member Can Enter The Consultation Room
   Given I have a hearing with a Panel Member
   And the Panel Member user has progressed to the Judge Waiting Room page for the existing hearing
-  And the hearing status changes to In Session
   When they enter the private consultation room
   Then they will be transferred to the private consultation room
```

Comment:
The wording needs to be changed.  with JoH it is just consultation room.  Private Meeting is for individuals, eg; LIP